book/doctrine.rst

```diff
@@ -979,126 +1060,66 @@ To relate the ``Category`` and ``Product`` entities, start by creating a
             </entity>
         </doctrine-mapping>
 
-First, since a ``Category`` object will relate to many ``Product`` objects,
-a ``products`` array property is added to hold those ``Product`` objects.
-Again, this isn't done because Doctrine needs it, but instead because it
-makes sense in the application for each ``Category`` to hold an array of
-``Product`` objects.
+Doctrine does not *require* that the "one" side of a one-to-many relationship
+hold a collection of its "many" related entities. But in the context of our
+application, it makes sense for each ``Category`` object to hold an
+array of ``Product`` objects. However, if we had decided against adding
+a ``$products`` property to the ``Category`` class, then the ``Product``
+entity's ``inversedBy`` metadata would not have been necessary.
 
 .. note::
 
-    The code in the ``__construct()`` method is important because Doctrine
-    requires the ``$products`` property to be an ``ArrayCollection`` object.
+    The code in the constructor is important because the ``$products`` property
+    must be an ``ArrayCollection`` object, rather than a traditional ``array``.
```

Comment:
It's not important that it is an `ArrayCollection`, but it must be a `Collection` implementation (for example, Doctrine will populate the property with a `PersistentCollection` instance by default when data are queried from the database).